### ISSUE TITLE
`osc lbl` is terribly inefficient

### ISSUE THREAD

**Issue by msirringhaus** [openSUSE/osc#813]
With big build logs, doing `osc lbl` takes *a long* time:
```
$ time isc lbl > /dev/null

real    0m34.623s
user    0m34.507s
sys     0m0.069s

$ time cat /var/tmp/build-root/standard-x86_64/.build.log > /dev/null

real    0m0.315s
user    0m0.001s
sys     0m0.079s
```

Same numbers when not piping to `/dev/null`.

**Comment by marcus-h** [comment]
On 2020-06-24 00:21:06 -0700, msirringhaus wrote:
> With big build logs, doing `osc lbl` takes *a long* time:
> ```
> $ time isc lbl > /dev/null
>
> real    0m34.623s
> user    0m34.507s
> sys     0m0.069s
>
> $ time cat /var/tmp/build-root/standard-x86_64/.build.log > /dev/null
>
> real    0m0.315s
> user    0m0.001s
> sys     0m0.079s
> ```
>
> Same numbers when not piping to `/dev/null`.
>
It should be fixed with commit 276d6e2439c8c53c182dbe785b038919e64da9f3
("Do not use the chardet module in util.helper.decode_it").

**Comment by lethliel** [comment]
Fix is released with osc 0.170.0
